eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/ManagementOutputLibrary.cs

```diff
@@ -337,46 +337,88 @@ private Dictionary<CSharpType, OperationSourceProvider> BuildOperationSources()
             {
                 foreach (var resourceMethod in metadata.Methods)
                 {
-                    ProcessLroMethod(resourceMethod.InputMethod, operationSources);
+                    if (resourceMethod.InputMethod.IsLongRunningOperation())
+                    {
+                        ProcessLroMethod(resourceMethod.InputMethod, operationSources);
+                    }
+                    else
+                    {
+                        ProcessNonLroMethod(resourceMethod.InputMethod, operationSources);
+                    }
```

Comment:
I did not get the point - OperationSources are only used in the response of LRO methods.
Why do we need to process non lro methods here in the first place?
I think we need some comments here.